### PR TITLE
feat: update logo shown on frontend

### PR DESCRIPTION
**Pull request by makram93**
I have updates the torch requirements to `torch>=1.10.2` and the logo looks like this:

image

Also, we get userwarnings from Jina which is unwanted as shown here:
image
But now it no more shows if the program is run via `jina-now start`. `jina now start` will still give this error as the control is first with jina and then with NOW
image

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
     ]
 sys_platform = platform.system().lower()
 # if sys_platform == 'darwin':
-torch_requirement = ['torch==1.10.2']
+torch_requirement = ['torch>=1.10.2']
```
Comment (joschkabraun) [inline setup.py:38]: Do we support torch>=1.10.2?

**Review by joschkabraun — APPROVED** [review]
LGTM

**Review by joschkabraun — APPROVED** [review]

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-torchvision==0.11.3
+torchvision>=0.11.3
```
Comment (florian-hoenicke) [inline requirements.txt:1]: should be handled in a different pr next time

**Review by florian-hoenicke — CHANGES_REQUESTED** [review]
Great initiative updating the logo.
However, the logo should be jpg or SVG.

**Review by florian-hoenicke — CHANGES_REQUESTED** [review]
why are the banners added?

**Comment by makram93** [comment]
> Great initiative updating the logo. However, the logo should be jpg or SVG.

Are you referring to the `favicon.png`? I cross-referenced with other repo - docarray and jina, both of which have favicon and banner in png format.



> why are the banners added?

It is not important but to have uniformity with other repos such as [here](https://github.com/jina-ai/jina/blob/master/docs/conf.py#L168), I added it for NOW. Usage of this can be found [here](https://github.com/wpilibsuite/sphinxext-opengraph/blob/main/README.md#:~:text=ogp_image,defined%20by%20ogp_site_name.).

File: requirements.txt (hunk 2 above)
Comment (makram93) [reply requirements.txt:1]: I will revert this for this PR

File: now/run_all_k8s.py [hunk 3]
```diff
@@ -58,7 +58,7 @@ def run_k8s(os_type: str = 'linux', arch: str = 'x86_64', **kwargs):
             **kwargs,
         )
         with tempfile.TemporaryDirectory() as tmpdir:
-            docker_frontend_tag = '0.0.7'
+            docker_frontend_tag = '0.0.8-dev-update-logo'
```
Comment (florian-hoenicke) [inline now/run_all_k8s.py:61]: I think this should be just 0.0.8.
Or correct me please @joschkabraun if I'm mistaken.

**Review by florian-hoenicke — CHANGES_REQUESTED** [review]

**Review by florian-hoenicke — APPROVED** [review]
LGTM

File: now/run_all_k8s.py (hunk 3 above)
Comment (makram93) [reply now/run_all_k8s.py]: I have updated the version. It should be `0.0.8` now

**Pull request merged**